sampler_remote.go

```diff
@@ -200,6 +200,7 @@ func (s *RemotelyControlledSampler) updateSamplerViaUpdaters(strategy interface{
 			return err
 		}
 		if sampler != nil {
+			s.logger.Debugf("sampler updated: %+v", sampler)
```

Comment:
The logger is a debug logger - the stringer on `sampler` won't be called unless the log level is set to debug. The default poll interval is 1 minute.